projects/plugins/boost/package.json

```diff
@@ -0,0 +1,116 @@
+		"test:env-start": "docker-compose -f docker/boost-test/docker-compose.yml up -d",
+		"test:env-stop": "docker-compose -f docker/boost-test/docker-compose.yml down",
+		"test:env-logs": "docker-compose -f docker/boost-test/docker-compose.yml logs -f",
+		"test:wp-cli": "docker run --env-file docker/boost-test/.env --rm --volumes-from boost-test_wordpress_1 --network container:boost-test_wordpress_1 -w /var/www/html --entrypoint /bin/bash wordpress:cli",
+		"test:env-rm": "docker-compose -f docker/boost-test/docker-compose.yml rm -f -s",
+		"test:activate-wordpress": "npm run test:wp-cli -- -c 'wp core install --url=http://localhost:9002 --title=Jetpack-Boost-Test --admin_user=admin --admin_password=jetpackboostpassword --admin_email=info@example.com --skip-email'",
+		"test:deactivate-wordpress": "npm run test:wp-cli -- -c 'wp db reset --yes'",
+		"test:activate-plugins": "npm run test:wp-cli -- -c 'wp plugin activate jetpack-boost jetpack-boost-sidekick-test'",
+		"test:deactivate-plugins": "npm run test:wp-cli -- -c 'wp plugin deactivate jetpack-boost jetpack-boost-sidekick-test'",
+		"dev:start": "npm-run-all dev:env-start && sleep 5 && npm-run-all dev:activate-wordpress dev:activate-plugins",
+		"dev:stop": "npm run dev:env-stop",
+		"dev:shell": "docker-compose -f docker/boost-dev/docker-compose.yml exec wordpress /bin/bash",
+		"dev:env-pull": "docker-compose -f docker/boost-dev/docker-compose.yml pull",
+		"dev:env-start": "docker-compose -f docker/boost-dev/docker-compose.yml up -d",
+		"dev:env-stop": "docker-compose -f docker/boost-dev/docker-compose.yml down",
+		"dev:env-logs": "docker-compose -f docker/boost-dev/docker-compose.yml logs -f",
+		"dev:wp-cli": "docker run --env-file docker/boost-dev/.env --rm -u 33 --volumes-from boost-dev_wordpress_1 --network container:boost-dev_wordpress_1 -w /var/www/html --entrypoint /bin/bash wordpress:cli",
+		"dev:db-cli": "docker run --rm -u 33 -it --volumes-from boost-dev_wordpress_1 --network container:boost-dev_wordpress_1 -w /var/www/html --entrypoint /bin/bash wordpress:cli -c 'wp db cli'",
+		"dev:env-rm": "docker-compose -f docker/boost-dev/docker-compose.yml rm -f -s",
+		"dev:activate-wordpress": "npm run dev:wp-cli -- -c 'if ! wp core is-installed; then sleep 60 && wp core install --url=http://localhost:9001 --title=Jetpack-Boost-Dev --admin_user=admin --admin_password=jetpackboostpassword --admin_email=info@example.com --skip-email; fi'",
+		"dev:activate-plugins": "npm run dev:wp-cli -- -c 'wp plugin activate jetpack-boost jetpack-boost-sidekick-dev-local'",
+		"release:build": "./bin/generate-release-zip.sh",
+		"release:branch": "./bin/generate-release-branch.sh",
+		"release:delete": "./bin/delete-release-branch.sh",
+		"prepare": "husky install",
+		"clear-dist": "rm -rf app/assets/dist/*"
+	},
+	"private": true,
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/Automattic/jetpack-boost.git"
+	},
+	"author": "",
+	"license": "ISC",
```

Comment:
ISC?

Reply:
A permissive MIT-like license, but as a WP plugin, we're going with the GPL. Updated in 07d018431